templates/form-fields/recaptcha-v3-field.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Shows the `recaptcha` form field on job listing forms.
+ *
+ * This template can be overridden by copying it to yourtheme/job_manager/form-fields/recaptcha-field.php.
+ *
+ * @see         https://wpjobmanager.com/document/template-overrides/
+ * @author      Automattic
+ * @package     wp-job-manager
+ * @category    Template
+ * @version     1.31.1
```

Comment:
```suggestion
 * @version     $$next-version$$
```

Reply:
done in 9f1c6217200aa0e65150ca94a576074a2f3c89e1